bundles/org.openhab.binding.nest/README.md

```diff
@@ -67,8 +67,9 @@ Now an OAuth 2.0 client is created and configured for using the SDM API by the b
 1. Create a new client on the "Credentials" page ([APIs & Services > Credentials](https://console.cloud.google.com/apis/credentials)):
     1. Click the "Create Credentials" button at the top of the page
     1. Choose "OAuth client ID"
-    1. As "Application type" choose "TVs and Limited Input devices"
+    1. As "Application type" choose "Web application"
     1. Give it a name so you can remember what it is used for (e.g. `Nest Binding SDM`)
+    1. Add "https://www.google.com" to the "Authorized redirect URIs"
     1. Click "Create" to create the client
     1. Copy and save the generated **Client ID** (e.g. `1046297811237-3f5sj4ccfubit0fum027ral82jgffsd1.apps.googleusercontent.com`) and **Client Secret** (e.g. `726kcU-d1W4RXxEJA79oZ0oG`) somewhere
```

Comment:
@wborn Can you confirm that these credentials are fake? If so, could we change them to be obviously fake ones?

Reply:
Yes they are fake. They resemble the real thing so it is clear what to enter. Does it trigger some kind of security scan?

Reply:
Yeah, exactly - it was an enterprise-internal vulnerability scan; apparently purely triggered by the fact that I have write access to this repo here and also use the same GitHub handle for professional contributions on other repos.